apps/client/components/ChatMessage.tsx

```diff
@@ -0,0 +1,9 @@
+
+
+export default function ChatMessage() {
```

Comment:
Maybe we should rename this to ChatMessageInput?

In the component below it's a bit confusing because it looks like it should render a message someone sent